package.json

```diff
@@ -15,7 +15,7 @@
     "postinstall": "npm run compile-contract-types",
     "e2e": "cypress open",
     "prepare": "husky install",
-    "prettier": "npx prettier -c **/*.{js,css,md,json,ts,tsx}",
+    "prettier": "npx prettier -c --write **/*.{js,css,md,json,ts,tsx}",
```

Comment:
I don't mean to seem like I'm peeking over your shoulder, but since lint-staged is set up to run Prettier, is this needed? :)

Reply:
No problem whatsoever! You're certainly right but since it was included in the current github actions i left it in.